lib/ProMotion/map/map_screen_module.rb

```diff
@@ -163,6 +163,9 @@ def annotation_view(map_view, annotation)
       if params[:right_accessory]
         view.rightCalloutAccessoryView = params[:right_accessory]
       end
+      if params[:detail_accessory]
+        view.detailCalloutAccessoryView = params[:detail_accessory]
```

Comment:
Not really. it actually kind of works using `UIImageView.alloc.initWithImage(UIImage.imageNamed("Default-568h")) `as the view
<img width="320" alt="screen shot 2016-04-15 at 18 22 12" src="https://cloud.githubusercontent.com/assets/572843/14575463/50cdd070-0338-11e6-9827-dba2085e69d7.png">